src/shapes/object.class.ts

```diff
@@ -122,15 +125,15 @@ export class FabricObject extends ObjectGeometry {
    * @type String
    * @default rgb(178,204,255)
    */
-  cornerColor: string | null;
+  cornerColor: string;
 
   /**
    * Color of controlling corners of an object (when it's active and transparentCorners false)
    * @since 1.6.2
    * @type String
    * @default null
    */
-  cornerStrokeColor: string | null;
+  cornerStrokeColor: string;
```

Comment:
empty string has the same effect of null, so we can reduce to string and keep it compatible with the embedded types of ctx.fillStyle and ctx.strokeStyle

Reply:
good
I came across that in #8400  as well